model/authorizations/v1/resource_review_request_type.model

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright (c) 2019 Red Hat, Inc.
+Copyright (c) 2021 Red Hat, Inc.
```

Comment:
Don't change dates of copyright notices. The date should be the date of initial creation of the file.